lib/iris/cube.py

```diff
@@ -1615,10 +1615,21 @@ def shape(self):
 
     @property
     def dtype(self):
+        """
+        The data type of the values in data array of this
+        :class:`~iris.cube.Cube`.
+
+        """
         return self._data_manager.dtype
 
     @property
     def fill_value(self):
+        """
+        The fill value for the masked data array of this
+        :class:`~iris.cube.Cube`. Will be ``None`` if the data array is
+        not masked.
```

Comment:
@dkillick The data of the cube doesn't need to be masked for the cube to have a `fill_value`. This allows users to control any fill value that is saved to a file format, for example.

A `fill_value` of `None` simply means that no fill value has been explicitly specified. This means that if the data is masked, then the fill value of the `cube.data` returned to the user will be the default `numpy` fill value for the `dtype` of the data.